libkbfs/folder_branch_ops.go

```diff
@@ -1683,7 +1683,8 @@ func isRecoverableBlockErrorForRemoval(err error) bool {
 }
 
 func isRetriableError(err error, retries int) bool {
-	recoverable := isRecoverableBlockError(err)
+	_, isEXCLOnUnmergedError := err.(EXCLOnUnmergedError)
+	recoverable := isRecoverableBlockError(err) || isEXCLOnUnmergedError
```

Comment:
super minor optimization: put `isEXCLOnUnmergedError` first to avoid a function call if it's already true.